apps/admin_app/assets/js/components/App.tsx

```diff
@@ -1,11 +1,24 @@
 import * as React from "react";
-import { Hello } from "./Hello";
-
-class App extends React.Component {
+// import { Hello } from "./Hello";
+import {PromotionsForm} from "./CreatePromotions"
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import {Promotions} from "./Promotions"
+class App extends React.Component<{},any> {
+  constructor(props){
+    super(props)
+    this.state={hello:"hi"}
```

Comment:
why this dummy state? Is it needed? Also why any in the state declaration?

Reply:
@SakinaBoriwala21 could you please let us know your thought process here?

Reply:
@VivekNayyar @ashish173 It's not at all needed, I was checking something and I forgot to remove it later on.